sale_require_po_doc/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Orders Require PO or Sales Documentation",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Sales",
+    "depends": [
+        "sale_management", "purchase"
```

Comment:
The purchase is not required for this feature.
```suggestion
        "sale_management"
```